hy/models/complex.py

```diff
@@ -30,3 +30,5 @@ class HyComplex(HyObject, complex):
     def __new__(cls, number, *args, **kwargs):
         number = complex(number)
         return super(HyComplex, cls).__new__(cls, number)
+
+_wrappers[complex] = HyComplex
```

Comment:
We can, but it will create some circular imports because then `hy/models/__init__.py` need to `import hy/models/complex.py` etc.. I'm not sure which way is better.